outputs.tf

```diff
@@ -1,11 +1,11 @@
 output "ses_domain_identity_arn" {
   description = "The ARN of the domain identity"
-  value       = aws_ses_domain_identity.default.arn
+  value       = var.domain_validation ? join("", aws_ses_domain_identity.default.*.arn) : null
```

Comment:
To stay consistent with our other modules, we typically just do the empty join pattern without the extra unneeded conditional. Can you update so value is just equal to `join("", aws_ses_domain_identity.default.*.arn)`. That will result in an empty string if `domain_validation` validation is false so accomplishes the same result.